### ISSUE TITLE
Not an Issue - just trying to make contact, I'm afraid!

### ISSUE THREAD

**Issue by smeech** [menawi/world-history#14]
Dear Omar

I've been reviewing your "espanso-dropdown-list-template" PR at https://github.com/espanso/hub/pull/85, and raised a number of issues that need fixing before we can merge.

I don't know if you've been notified of these (and it would be interesting to hear, as you're not the only person who hasn't responded!), so thought I'd try and make direct contact.

Let me know if you need any help with the amendments. Whilst I could probably make them, I think it's preferable if it's your own work.

With best wishes,

**Comment by menawi** [comment]
Hello Stephen, thank you for reaching out! I apologize for the delay as I
have been mired in a work site transfer process that's taking weeks.

I've also been taking on a larger load than usual at work so I can leave my
team without any loose strings hopefully.

I saw the earlier email and I've been meaning to make the changes submit
another PR very soon! Thank you for your follow up and let me know if you
would like to see similar packages


Best Regards,
Omar Elmenawi

CONFIDENTIALITY NOTICE:
This email and attached material are intended for the use of the individual
or organization to whom they are addressed and may not be distributed,
copied, or disclosed to other unauthorized persons. This material may
contain confidential and/or personal information subject to the provisions
of the Freedom of Information and Protection of Privacy Act, the Municipal
Freedom of Information and Protection of Privacy Act, and/or the Personal
Health Information Protection Act. If you receive this transmission in
error, please notify me immediately and delete this message. Do not email,
print, copy, distribute, or disclose this email or its contents further.
Thank you for your co-operation and assistance.

**Comment by smeech** [comment]
No problem and no rush!

Sorry about the delay from our end - it's been a couple of years since Federico left the project dormant, and the current team is only just getting to grips with things. As I don't really code much, I've focused on the documentation, hub, and user queries.
